src/app/shared/models/models.component.ts

```diff
@@ -530,11 +608,11 @@ export class ModelsComponent implements OnInit, OnDestroy {
     }
   };
 
-  onFavouriteDbClick = node => {
+  onFavouriteDbClick = (node) => {
```

Comment:
If in this area, might as well change this to `onFavoriteDbClick(node) { }` instead